clang/runtime/dpct-rt/include/dpct/bindless_images.hpp

```diff
@@ -365,9 +368,11 @@ template <class T, int dimensions> class bindless_image_wrapper {
     detail::get_img_mem_map(_img) = mem;
     auto ptr = data;
 #ifdef DPCT_USM_LEVEL_NONE
-    ptr = get_buffer(data).get_host_access().get_pointer();
-#endif
+    q.ext_oneapi_copy(get_buffer(data).get_host_access().get_pointer(),
```

Comment:
We should test macro when we use oneapi extensions. 
https://github.com/intel/llvm/blob/65cc0cfe809f52169b92e6bea782ddc626ba0a87/sycl/doc/extensions/experimental/sycl_ext_oneapi_copy_optimize.asciidoc

#ifdef SYCL_EXT_ONEAPI_COPY_OPTIMIZE
...
#endif // SYCL_EXT_ONEAPI_COPY_OPTIMIZE

Reply:
The ext_oneapi_copy only appeared in https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc, so I thinks the macro SYCL_EXT_ONEAPI_BINDLESS_IMAGES is enough.